t/storage.t

```diff
@@ -40,21 +53,21 @@ describe "HTFeed::Storage" => sub {
       my $volume = stage_volume(@_);
 
       my $storage = HTFeed::Storage::LocalPairtree->new(
-        volume => $volume);
+        volume => $volume,
+        config => config_for_class('HTFeed::Storage::LocalPairtree'));
 
       return $storage;
     }
 
     sub make_old_version_other_dir {
       my $storage = shift;
-
-      my $real_obj_dir = get_config('repository','obj_dir');
-      my $other_obj_dir = get_config('repository','other_obj_dir');
-      set_config($other_obj_dir,'repository','obj_dir');
+      my $real_obj_dir = $storage->get_storage_config('obj_dir');
```

Comment:
I think we want to change this to build a separate storage instance with the "other obj dir" (which we could get from our TempDirs object that just knows about the directories rather than fussing with the global config)

Reply:
I think the two instances are both getting config from `$tmpdirs` now with my second commit.